Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.64.0-slim-bullseye AS build
+FROM rust:1.75.0-slim-bookworm AS build
```

Comment:
Another one that should have been a different PR really :) Fwiw all our testing is with rustc 1.70 (as that's current in debian). But should be ok with current rustc